plasmapy/formulary/parameters.py

```diff
@@ -177,12 +182,11 @@ def Alfven_speed(
         Either the ion number density in units convertible to 1 / m**3,
         or the mass density in units convertible to kg / m**3.
 
-    ion : str, optional
+    ion : str
```

Comment:
For `Alfven_speed`, `density` can be either a mass density or a number density, so the argument `ion` should be optional since it's not needed when a mass density is provided.

Reply:
Ok, I took a stab at correcting this. Since it's optional, that means I had to put back the default value of `None`, so that the call to `mass_density` would work.